packages/cmc/src/cmc.canister.ts

```diff
@@ -25,14 +25,22 @@ export class CMCCanister extends Canister<CMCCanisterService> {
   }
 
   /**
-   * Returns conversion rate of ICP to Cycles
+   * Returns conversion rate of ICP to Cycles. It can be called as query or update.
+   *
+   * @param {Object} [params] - The parameters for the call.
+   * @param {boolean} [params.certified] - Determines whether the response should be certified (default: non-certified)
    *
    * @returns Promise<BigInt>
    */
-  public getIcpToCyclesConversionRate = async (): Promise<bigint> => {
-    const { data } = await this.service.get_icp_xdr_conversion_rate();
+  public getIcpToCyclesConversionRate = async ({
+    certified,
```

Comment:
We should maybe default to `true` for consistency (even if it's a breaking change)?

Reply:
I considered it, but then I noticed we were defaulting to false in other [functions](https://github.com/dfinity/ic-js/blob/215e4f4c395695ef152a9290410b172d1a70aafa/packages/cmc/src/cmc.canister.ts#L110) so I kept it like this. wdyt, should we enforce `true` for this one? At that point, does it still make sense to expose the parameter?

Reply:
Ahhh nvm my comment. I opened my code, the default value is set one level up:

```
protected caller = ({ certified = true }: QueryParams): T =>
    certified ? this.certifiedService : this.service;
```

So we are good with `{ certified }: QueryParams = {}`.